File: code.leetcode/string/easy/e58.最后一个单词的长度.cpp
class Solution
{
public:
    int lengthOfLastWord(string s)
    {
        for (int i = s.size() - 1; i >= 0; --i)
        {
            if (s[i] != ' ')
            {
                for (int j = i; j >= 0; --j)
                {
                    if (s[j] == ' ')
                    {
                        return i - j;
                    }
                    else if (j == 0)
                    {
                        return i + 1;
                    }
                }
            }
        }
        return 0;
    }
};

File: code.leetcode/string/easy/e58.最后一个单词的长度.py
class Solution(object):
    def lengthOfLastWord(self, s):
        i = len(s) - 1
        while i >= 0:
            if s[i] != ' ' :
                j = i
                while j >= 0:
                    if s[j] == ' '  :
                        return i - j
                    if j == 0 :
                        return i + 1
                    j -= 1
            i-=1		
			
		
		

        return 0